src/Scene/NodeProcess.js

```diff
@@ -135,6 +197,67 @@ define('Scene/NodeProcess',
         }
     };
 
+    function updateNodeImagery(quadtree, node) {
+        var promises = [];
+
+        for (var i=0; i<quadtree.colorLayers.length; i++) {
+            var layer = quadtree.colorLayers[i];
+
+            if (layer.tileInsideLimit(node, layer)) {
+                var args = {layer: layer, destination: 1 };
+                promises.push(quadtree.interCommand.request(args, node, refinementCommandCancellationFn));
+            }
+        }
+
+        return Promise.all(promises).then(function(colorTextures) {
+            var textures = [];
+            for (var j=0; j<colorTextures.length; j++) {
+                textures = textures.concat(colorTextures[j]);
+            }
+            node.setTexturesLayer(textures, 1);
```

Comment:
Wouldn't node.setTexturesLayer(colorTextures, 1) achieve the same result? Without the need to create a new array.

I think we should also split setTexturesLayer into setElevationTextures and setColorTextures to get rid of these annoying "0" and "1".